umi_gripper/umi_gripper.xml

```diff
@@ -0,0 +1,141 @@
+<mujoco model="umi_model">
+
+  <compiler angle="radian" meshdir="assets/" texturedir="assets/" />
```

Comment:
Minor nit, can you remove the forward slash at the end of the meshdir and texturedir?